8_viz/src/plot_lake_obs_counts.R

```diff
@@ -31,20 +31,15 @@ plot_lake_obs_counts <- function(
     site_counts_mat <- matrix(
       NA, nrow=length(mins_obs_per_date), ncol=length(mins_obs_dates),
       dimnames=list(min_obs_per_date=mins_obs_per_date, min_obs_dates=mins_obs_dates))
-    for(min_obs_per_date in mins_obs_per_date) {
+    site_counts_mat['0', '0'] <- obs_counts %>% pull(site_id) %>% unique() %>% length()
```

Comment:
I revised how I calculated the lone triangle and circle just to make sure I got it right - I think this code is more straightforward, but it produces the same numbers as before